model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: 'Semantic Convention for pipeline runs'
+    stability: development
```

Comment:
could you please detailed description you have in markdown for this span into the yaml? 

See https://github.com/open-telemetry/semantic-conventions/blob/08728eb08de176cb5c0616022feabb5018741e00/model/database/spans.yaml#L110 as an example.

It helps us keep all these notes at least available for the tooling.

Also, same concern on the span.cicd.pipeline.task.internal

Reply:
Done in e41f4f520fb814e4545591cfa7c27ac6c47476e0.

There was no markdown for cicd.pipeline.task to be moved to the yaml.
I will keep it in mind if we decide to add some later.